### ISSUE TITLE
SignalPlot Linestyle.None is still visible

### ISSUE THREAD

**Issue by nassaleh** [ScottPlot/ScottPlot#1797]
## Bug Report

**Issue:** (Bug description goes here)
When plotting a Signal plot, setting the line style to LineStyle.None has no effect.

**Reproducing:** (What do others have to do to reproduce the bug?)

```cs
// a code sample may improve communication
var examplePlot = new ScottPlot.Plot();
double[] values = DataGen.Sin(100);`
var signalPlot = examplePlot.AddSignal(values, 1);
signalPlot.LineStyle = LineStyle.None

```

### System Details
* **ScottPlot Version:** 4.1.36
* **Operating System:** Windows 10
* **Application Type:** WinForms
* **.NET Version:** NET 6.0

**Comment by nassaleh** [comment]
Workaround is to set the LineWidth to 0;

**Comment by swharden** [comment]
Thanks for reporting this @nassaleh! Your workaround is good, but I fixed the primary issue in #1817 so this will be less confusing for future users.
